app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,58 @@
+class DriversController < ApplicationController
+  def index
+    @drivers = Driver.all
+  end
+
+  def show
+    @driver = Driver.find_by(id: params[:id])
+
+    if @driver.nil?
+      head :not_found
+    end
+  end
+
+  def new
+    @driver = Driver.new
+  end
+
+  def create
+    filtered_driver_params = driver_params()
+    @driver = Driver.new(filtered_driver_params)
+
+    if @driver.save
+      redirect_to drivers_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @driver = Driver.find_by(id: params[:id])
+  end
+
+  def update
+    @driver = Driver.find(params[:id])
+
+    if @driver.update(driver_params)
```

Comment:
It's interesting to me that in `create` you store the strong params in an intermediate local variable, but in `update` you pass it directly into the model. Using the local variable is fine if it makes the code clearer to you, but you should follow the same pattern in both places.